pkg/cmd/list.go

```diff
@@ -277,8 +279,8 @@ func (w *WhoCan) validate(action Action) error {
 
 // Check checks who can perform the action specified by WhoCanOptions and returns the role bindings that allows the
 // action to be performed.
-func (w *WhoCan) Check(action Action) (roleBindings []rbac.RoleBinding, clusterRoleBindings []rbac.ClusterRoleBinding, err error) {
-	err = w.validate(action)
+func (w *WhoCan) Check(ctx context.Context, action Action) (roleBindings []rbac.RoleBinding, clusterRoleBindings []rbac.ClusterRoleBinding, err error) {
```

Comment:
I'd not change the method interface because we do not use the context.Context at all. We can pass context.Background() directly to the client.